docs/articles/nunit/getting-started/installation.md

```diff
@@ -1,43 +1,149 @@
+The results will be exactly the same as if you had used Visual Studio.
+
+Rider has built-in support for NUnit, but are using the same template, so you get the full result.
+
+## I'm using Rider as my development tool
+
+This is the same procedure as for Visual Studio, `Add New Project`, select the `NUnit Test Project` template and follow the prompts.
+The results will be exactly the same as if you had used Visual Studio.
+
+## I prefer to do this using the command line or any of the built-in terminals
+
+From your solution folder, run `dotnet new nunit -o TestProject1` to create a new NUnit test project in the folder `TestProject1`.
+The results will be exactly the same as if you had used Visual Studio.
+
+## Examples of what you get
+
+All of the above will create a new project with the following structure, after updating the packages to the latest versions (of Sept 2023):
+
+```csproj
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
+        <PackageReference Include="NUnit" Version="3.13.3" />
+        <PackageReference Include="NUnit3TestAdapter" Version="4.5.0" />
+        <PackageReference Include="NUnit.Analyzers" Version="3.7.0">
+          <PrivateAssets>all</PrivateAssets>
+          <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+        <PackageReference Include="coverlet.collector" Version="6.0.0">
+          <PrivateAssets>all</PrivateAssets>
+          <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+     </ItemGroup>
+
+</Project>
+
+```
+
+It will create a `UnitTest1.cs` file with the following contents:
+
+```cs
+namespace TestProject1;
+
+public class Tests
+{
+    [SetUp]
+    public void Setup()
+    {
+    }
+
+    [Test]
+    public void Test1()
+    {
+        Assert.Pass();
+    }
+}
+```
+
+and a `Usings.cs` file:
+
+```cs
+global using NUnit.Framework;
+```
+
+## Adding NUnit to an existing test project
+
+If you need to update an existing test project to use NUnit, it may a standard class lib you want to convert to a test project, or you want to add NUnit to a project that already has XUnit or MSTest, you can do that by adding the itemgroup in the csproj above to your project file, and then the appropriate `using NUnit.Framework;` in your code.
+
+Also note that multiple different test frameworks (XUnit, MSTest, NUnit) can co-exist in the same test project, preferably in different files to avoid name clashes, see [this old blog post](https://devblogs.microsoft.com/devops/part-2using-traits-with-different-test-frameworks-in-the-unit-test-explorer/) for details.
```

Comment:
```suggestion
Also note that multiple different test frameworks (XUnit, MSTest, NUnit) can co-exist in the same test project, preferably in different files to avoid name clashes. See [this blog post](https://devblogs.microsoft.com/devops/part-2using-traits-with-different-test-frameworks-in-the-unit-test-explorer/) for details.
```